kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF/mii-update3-d1-JRF-v2-ds.yaml

```diff
@@ -122,8 +122,17 @@ spec:
   # roll of your domain's WebLogic Server pods.
   restartVersion: '1'
 
+  # Changes to this field cause the operator to repeat its introspection of the
+  #  WebLogic domain configuration.
+  introspectVersion: '1'
+
   configuration:
 
+    # Operator will attempt to use the online update to the running domain without
```

Comment:
Suggested rerewording:

# If set to 'true' and an introspector job detects that
# WebLogic configuration changes for an already running
# domain are all dynamic, then the operator will attempt
# to use an online update to update WebLogic configuration
# instead of restarting all WebLogic server pods.

Reply:
Thanks. updated.